src/compiler/diagnosticMessages.json

```diff
@@ -3118,7 +3118,7 @@
         "category": "Error",
         "code": 18000
     },
-    "The path in an 'extends' options must be relative or rooted.": {
+    "The paths in an 'extends' option must be relative or rooted.": {

```

Comment:
Isn't there only single inheritance of configurations, in the end? Can't there be only one "extends" path? So "paths" should be singular "path"?